lib/navigation/00.froxlor.main.php

```diff
@@ -229,6 +229,11 @@
 					'label' => $lng['domains']['ssl_certificates'],
 					'required_resources' => 'domains'
 				),
+				array(							// DF8OE
+					'url' => 'admin_domains.php?page=dkim_keys',
+					'label' => $lng['domains']['ssl_certificates'],
```

Comment:
wrong label

Reply:
also, missing menu-entry for customers

Reply:
please also add `'show_element' => ...` to show this only when DKIM is enabled

Reply:
1) and 2): yes: not finished yet. 3) I will add.